lib/Client.js

```diff
@@ -42,11 +42,12 @@ Client.auth = function (options, cb) {
  * @param  {String}   client Client Token
  * @param  {Function} cb     (err, new token, full response body)
  */
-Client.refresh = function (access, client, cb) {
+Client.refresh = function (access, client, cb, requestUser) {
   const host = this.host || defaultHost
```

Comment:
Callback should always be the last argument.
If you want to have an optional argument you can use the trick to check the argument type

Reply:
I did this for backwards compatible reasons. Is it okay to change the order?

Reply:
If the parameter is not optional then it's a breaking change regardless of the order
If the parameter is optional, you can use the trick I mentioned to keep the callback at the end without making a breaking change
So I'd advise the second option